eslint-plugin/docs/rules/no-unmocked-core-modules.md

```diff
@@ -0,0 +1,22 @@
+# Disallow the use of unmocked node core module (no-unmocked-core-modules)
+
+Node modules are [automatically mocked by jest](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules). That means that if a file exists in the `__mocks__` directory next to `node_modules`, the mock will be returned instead of the actual module on a `require/import`.
```

Comment:
Is this rule automatically enforced? Or should this be part of the contributing guidelines?

Reply:
See my comment bellow.